channelz/service/service_test.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package grpc.credentials;
+
+option go_package = "google.golang.org/grpc/channelz/service";
```

Comment:
Can this go into `service_test`?  If not, please put it in a subdirectory so it doesn't get pulled into the production build.  (Even if it is eventually pruned out (?), it still feels wrong.)

Reply:
Yeah, that's why I wanted to keep it as a descriptor in the test file at first... I think I will move the info into the test file like before.